src/mmw/js/src/modeling/controls.js

```diff
@@ -57,29 +59,70 @@ var DrawControlView = ControlView.extend({
     }
 });
 
-var LandCoverView = DrawControlView.extend({
-    template: landCoverTmpl,
+var SummaryView = Marionette.ItemView.extend({
+    template: summaryTmpl,
+
+    modelEvents: {
+        'change:thumbValue': 'render'
+    },
 
     templateHelpers: function() {
+        var thumbValue = this.model.get('thumbValue'),
+            label = '',
+            summary = '';
+        if (thumbValue) {
+            label = models.getHumanReadableLabel(thumbValue);
+            summary = models.getHumanReadableSummary(thumbValue);
+        }
         return {
-            label: models.getHumanReadableLabel
+            label: label,
+            summary: summary
         };
+    }
+});
+
+var ModificationsView = DrawControlView.extend({
+    ui: _.defaults({
+        thumb: '.thumb'
+    }, DrawControlView.prototype.ui),
+
+    regions: {
+        summaryRegion: '.summary-region'
+    },
+
+    events: _.defaults({
+        'mouseenter @ui.thumb': 'setThumbValue',
+        'mouseleave @ui.thumb': 'clearThumbValue'
+    }, DrawControlView.prototype.events),
+
+    templateHelpers: {
+        labelFn: models.getHumanReadableLabel
+    },
+
+    setThumbValue: function(event) {
```

Comment:
I think there are some good reasons not to use the word `event` as an parameter or variable name.
https://stackoverflow.com/questions/1510094/is-event-a-reserved-word-in-javascript
I think we generally go with `e` or `evnt`.
Others may feel differently though.